src/cc2olx/qti.py

```diff
@@ -321,7 +334,7 @@ def parse_qti(self):
         root = tree.getroot()
 
         # qti xml can contain multiple problems represented by <item/> elements
```

Comment:
Is this comment still relevant?

Reply:
Yes, we changed the code to consider all sections instead of just one with `root_section` identifier. Comment doesn't relate to identifiers, so nothing changed related to the comment, we still can have multiple `<item/>` elements.